script/_otbr

```diff
@@ -30,6 +30,7 @@
 readonly OTBR_TOP_BUILDDIR="${BUILD_DIR}/otbr"
 readonly OTBR_TOP_SRCDIR="$PWD"
 readonly OTBR_OPTIONS="${OTBR_OPTIONS:-}"
+readonly OTBR_POSIX_VENDOR_CLI="${OTBR_POSIX_VENDOR_CLI:-}"
```

Comment:
Does these need to be specific for CLI?  Couldn't we use this technique as a generic platform for extending the OTBR?

Reply:
I'm thinking maybe rename to `OTBR_POSIX_VENDOR_SRCDIR`